help/markdown/fake-dotnetcore.md

```diff
@@ -9,7 +9,7 @@ Just install the corresponding package for your system:
 choco install fake -pre
 ```
 
-now you can use 
```

Comment:
Just a note for your editor settings, whitespace at line endings is significant for markdown files (at least AFAIK)

Reply:
I checked my settings in Code and trim white space is turned on. Didn't know it is significant for MD.